Assignment/Program.c

```diff
@@ -9,33 +9,113 @@
 #include <stdio.h>
 #include <io.h>
 #include <fcntl.h>
-int main () {
-   int a = _setmode (_fileno (stdout), _O_U16TEXT);
-   wchar_t whitePieces[8] = { 0x2656, 0x2658, 0x2657, 0x2655, 0x2654, 0x2657, 0x2658, 0x2656 };
-   wchar_t blackPieces[8] = { 0x265C, 0x265E, 0x265D, 0x265B, 0x265A, 0x265D, 0x265E, 0x265C };
-   wchar_t whitePawn = 0x2659;
-   wchar_t blackPawn = 0x265F;
-   wprintf (L"%lc", 0x250F); //top
-   for (int i = 0; i < 7; i++) wprintf (L"%lc%lc%lc%lc", 0x2501, 0x2501, 0x2501, 0x2533);
+
+void PrintChessBoard (FILE* output);
+int CompareFiles (const char* outputFile, const char* referenceFile);
+
+void PrintChessBoard (FILE* output) {
+   wchar_t whitePieces[8] = { 0x2656, 0x2658, 0x2657, 0x2655, 0x2654, 0x2657, 0x2658, 0x2656 },
+      blackPieces[8] = { 0x265C, 0x265E, 0x265D, 0x265B, 0x265A, 0x265D, 0x265E, 0x265C },
+      whitePawn = 0x2659, blackPawn = 0x265F;
+   // top border
+   wprintf (L"%lc", 0x250F);

```

Comment:
Instead of printing and storing individual characters, use methods like wprintf(L"%s", top order/bottom order) and fputws(output, string)